webapp/sources/rudder/rudder-web/src/main/elm/healthcheck/sources/DataTypes.elm

```diff
@@ -0,0 +1,32 @@
+module DataTypes exposing (..)
+
+import Http exposing (Error)
+type alias Healthcheck = List Check
+
+type TabMenu
+    = General
+    | Details
+
+type SeverityLevel
+    = Critical
+    | Warning
+    | CheckPassed
+
+type alias Check =
+    { msg   : String
+    , level : SeverityLevel
+    }
+
+type alias Model =
+    { contextPath : String
+    , healthcheck  : Healthcheck
+    , tab         : TabMenu
+    , showChecks  : Bool
+    , healthStatus: SeverityLevel
```

Comment:
I don't think you need a healthStatus attribute

Reply:
I refactor the code to use it instead of compute the higher secrurity level in View

Reply:
I prefer to compute than having a value that could be different ;)